handbook/engineering/cloud/index.md

```diff
@@ -6,6 +6,26 @@ The cloud team owns all work that is necessary to build, secure, scale, and oper
 
 1. Any user or organization can use Sourcegraph Cloud with both the private and public code that they care about in a secure way.
 
+## Milestones
+
+1. Any user can add public code
+2. Organizations can use Sourcegraph with up to 50 repos or 100k LoC of private repositories for free
```

Comment:
I think this milestone should be split up between allowing private code on Sourcegraph.com and how we decide if its free. They seem quite independent and bundling them together will make it harder to measure success if we don't deliver both.

Reply:
Agreed, as per our live discussion today. I'll change.